FordLabs/PeopleMover
# Resolve flaky cypress people and product tests

## Issue
Connects N/A

## What was done
- [x] Updated the cypress people and product tests to include additional requirements for what is expected to be saved to the database when creating a new product or person.
- [x] Fix flaky Person cypress tests 
- [x] Fix flaky product test

On some of the forms when creating a role, location, or product tag, apparently it's necessary to add a wait for that new item to be created before continuing on with the cypress test.

## How to test
1. Run the cypress tests for people and product and ensure that there is adequate testing for what is being saved in the database.
